packages/shared/lib/db/database.ts

```diff
@@ -31,6 +32,41 @@ export class KnexDatabase {
         this.knex = knex(dbConfig);
     }
 
+    /**
+     * Not enabled by default because shared is imported by everything
+     */
+    enableMetrics() {
+        if (process.env['CI']) {
+            return;
+        }
+
+        const pool = this.knex.client.pool as Pool<any>;
+        const acquisitionMap = new Map<number, number>();
+
+        setInterval(() => {
+            metrics.gauge(metrics.Types.DB_POOL_USED, pool.numUsed());
+            metrics.gauge(metrics.Types.DB_POOL_FREE, pool.numFree());
+            metrics.gauge(metrics.Types.DB_POOL_WAITING, pool.numPendingAcquires());
+        }, 1000);
```

Comment:
It's not really precise, but 1s should give us a trend without spamming the metric, lmk